server/controllers/home.js

```diff
@@ -5,7 +5,7 @@ const utils = require('./utils');
 
 module.exports = {
   GET(req, res) {
-    if (!req.session) {
+    if (!req.session.user) {
```

Comment:
Pretty sure `!req.session` is fine since `req.session` will be `null` as per your middleware. But this works too, since `req.session.user` will be undefined in this case.

Reply:
the middleware now passes an object with .user defaulting to null, so this check is necessary.

Reply:
Oh got it, okay.